config/routes.rb

```diff
@@ -59,6 +59,7 @@
     VERSION_CONSTRAINT = /[\w\.\-]+/
 
     get '/maintenance/stats/enqueue/:platform/:name', to: 'maintenance_stats#enqueue'
+    get '/maintenance/stats/begin/:platform/:name', to: 'maintenance_stats#begin_watching'
```

Comment:
We should add the platform and name constraints here that the other routes are using.

Reply:
Can't believe I forgot! The reason why I put them below the constraints was to use them, just forgot to put them on...